CHANGELOG.rst

```diff
@@ -876,7 +876,7 @@ Bug Fixes
 Features
 ~~~~~~~~
 
-* **Renku CLI**: support moving files between datasets with ``renku mv`` (`CLI documentation <https://renku-python.readthedocs.io/en/latest/commands.html#module-renku.cli.move>`__).
+* **Renku CLI**: support moving files between datasets with ``renku mv`` (`CLI documentation <https://renku-python.readthedocs.io/en/latest/reference/commands.html#module-renku.cli.move>`__).
```

Comment:
the renku-python docs should be linked through the main renku docs, e.g. https://renku.readthedocs.io/en/latest/renku-python/docs/reference/commands.html#module-renku.ui.cli.move

Reply:
Adjusted, thanks!